packages/tradershub/src/features/cfd/modals/MT5PasswordModal/ButtonGroups/SuccessButtonGroup.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
```

Comment:
Same here if reusable 
We need to remove it from MT5PasswordModal folder

Reply:
same as above, will be refactored in dxtrade improvement since they are using same components